### DISCUSSION TITLE
Re-use basemap across projects

### DISCUSSION THREAD

**Discussion by maxhenschell** [opengisch/QField#2501] [Q&A]
Hi all-

I have created a base map using a Map Theme that I would like to use across multiple QField projects. Is that possible without re-building the base map for each project when creating it in QField Sync? And could multiple projects then point to that geopackage containing the base map on the mobile device? I am using QGIS 3.22.5 with QField Sync 4.0.0-BETA17 on Win10, and the QField Dev version currently available on Google Play. Thanks!

**Comment by ronanren** [comment]
Hi, you can use same basemap (gpkg file) for all projects on QField, here is the explanation [https://qfield.org/docs/prepare/outside-layers.html](url).

You need to put your basemap in "basemaps" folder in QField files on Android.
For me, I drag my basemap in QGis project and the directory of the basemap on my computer, I put it in the settings "Localized Data Paths".

Best regards,
Ronan

**Reply by suricactus** [reply]
Note this will change a bit with the new 2.0 major version that is coming. We are currently working on the docs and they will be updated accordingly.
